Gemfile

```diff
@@ -2,4 +2,8 @@
 # If you modify this file manually all comments and formatting will be lost.
 
 source "https://rubygems.org"
-gemspec :name => "logstash-core"
+gem "logstash-core", "2.0.0.dev", :path => "."
+gem "ci_reporter", "1.9.3", :group => :development
+gem "simplecov", :group => :development
+gem "coveralls", :group => :development
+gem "rspec", "~> 2.14.0", :group => :development
```

Comment:
hm. the Gemfile does not depend on `logstash-devutils` but does depends on `rspec` for running specs from the top level I think it's good to declare `rspec` as a dependency here. In other words, its not because some gems you depend on depend themselves on `rspec` that you should not declare this dependency.